fairing/kubernetes/manager.py

```diff
@@ -61,7 +61,7 @@ def secret_exists(self, name, namespace):
         secret_names = [secret.metadata.name for secret in secrets.items]
         return name in secret_names
 
-    def watch_service_for_external_ip(self, name, namespace, selectors=None):
+    def get_service_external_ip(self, name, namespace, selectors=None):
```

Comment:
since it is returning an URL, a better name would be get_service_external_endpoint?